txn_writer.go

```diff
@@ -21,15 +21,43 @@ import (
 // Example string to be written to the manifest file
 // if no dependencies are found in the project
 // during `dep init`
-const exampleToml = `
-# Example:
+const exampleTOML = `
+## EXAMPLES & DOCS - safe to delete!
+#
+## Dependencies define constraints on how dependent projects should be
+## incorporated into Gopkg.lock. They are respected by dep whether
+## this project is the current project, or if it's a dependency.
 # [[dependencies]]
```

Comment:
Perhaps each line should precede its documentation. It might be clearer that way.

```
# [[dependencies]]
## The dependencies section defines constraints ...
# name = "..."
## The required name field specifies the root...
```